### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by Astargh** [johngodley/redirection#640]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 7.0.22-0ubuntu0.16.04.1
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Cookie nonce is invalid
Action: redirection/v1/setting GET
Code: 403 Forbidden
Raw: {"code":"rest_cookie_invalid_nonce","message":"Cookie nonce is invalid","data":{"status":403}}
```

**Comment by johngodley** [comment]
Can you refresh the page and try again?

**Comment by johngodley** [comment]
You can also try the 3.1 beta:

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released 3.1. I will close this ticket for now, but please reopen if the problem persists
